packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import { IImageHandledProps, IImageMangedClasses, IImageUnhandledProps } from "./image.props";
+import { IImageClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts";
+
+/**
+ * TODO #239 #240: Hook viewports values into Styles
+ */
+export interface IBreakpoints {
+    vp1: number;
+    vp2: number;
+    vp3: number;
+    vp4: number;
+    vp5: number;
+    vp6: number;
+}
+
+const breakpoints: IBreakpoints = {
+    vp1: 0,
+    vp2: 540,
+    vp3: 768,
+    vp4: 1084,
+    vp5: 1400,
+    vp6: 1779
+};
+
+/* tslint:disable-next-line */
+class Image extends Foundation<IImageHandledProps & IManagedClasses<IImageClassNameContract>, React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
+    protected handledProps: HandledProps<IImageHandledProps & IManagedClasses<IImageClassNameContract>> = {
+        managedClasses: void 0,
+        alt: void(0),
+        itemScope: void(0),
+        round: void(0),
+        sizes: void(0),
+        src: void(0),
+        srcSet: void(0),
+        vp1: void(0),
+        vp2: void(0),
+        vp3: void(0),
+        vp4: void(0),
+        vp5: void(0),
+        vp6: void(0)
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLImageElement | HTMLPictureElement> {
+        if (!this.props.src && !this.props.vp1) {
+            return;
+        }
+
+        if (this.props.src) {
+            return (
+                <img
+                    {...this.unhandledProps()}
+                    className={this.generateClasses()}
+                    alt={this.props.alt}
+                    src={this.props.src}
+                />
+            );
+        } else {
+            return (
+                <picture
+                    {...this.unhandledProps()}
+                    className={this.generateClasses()}
+                >
+                    {this.generateSourceElement(6)}
+                    {this.generateSourceElement(5)}
+                    {this.generateSourceElement(4)}
+                    {this.generateSourceElement(3)}
+                    {this.generateSourceElement(2)}
+                    {this.generateSourceElement(1)}
+                    <img srcSet={this.props.vp1} src={this.props.vp1} alt={this.props.alt} className={this.generateClasses(true)}/>
```

Comment:
@eljefe223 I think you want this up on the image on line 56 instead, along with sizes? This would probably conflict with the source elements being generated here.